xml/System.Data.SqlClient/SqlParameter.xml

```diff
@@ -242,9 +242,7 @@
 Parameter = new SqlParameter("@pname", (object)0);  
 ```  
   
- If you do not perform this conversion, the compiler assumes that you are trying to call the `SqlParameter` (`string`, `SqlDbType`) constructor overload.  
-  
-   
+If you do not perform this conversion, the compiler assumes that you are trying to call the <xref:System.Data.SqlClient.SqlParameter.#ctor(System.String,System.Data.SqlDbType)> constructor overload.  
```

Comment:
I believe the warning you're getting is because of this: 

```suggestion
If you do not perform this conversion, the compiler assumes that you are trying to call the <xref:System.Data.SqlClient.SqlParameter.%23ctor(System.String,System.Data.SqlDbType)> constructor overload.  
```